README.md

```diff
@@ -2302,6 +2303,34 @@ Server: spray-can/1.3.3
 
 ```
 
+
+## GET /api/workflows/:version/backends
+
+This endpoint returns a list of the backends supported by the server.
+
+cURL:
+```
+$ curl http://localhost:8000/api/workflows/v1/backends
+```
+
+HTTPie:
+```
+$ http http://localhost:8000/api/workflows/v1/backends
+```
+
+Response:
+```
+HTTP/1.1 200 OK
```

Comment:
TOL: Why do we include this header junk in the docs? It doesn't seem useful. Full disclosure: I copy/pasted from elsewhere